airtime_mvc/application/common/PodcastManager.php

```diff
@@ -64,7 +64,7 @@ protected static function _findUningestedEpisodes($podcast, $service) {
             //  episodes in the list of episodes to ingest, don't skip this episode - we should try to ingest the
             //  most recent episode when the user first sets the podcast to automatic ingest.
             // If the publication date of this episode is before the ingest timestamp, we don't need to ingest it
-            if ((empty($ts) && !empty($episodes)) || strtotime($episodeData["pub_date"]) < strtotime($ts)) {
```

Comment:
What timezone is the publication date in? Is that the same timezone the auto-ingest timestamp is in? (Are they always the same?)

What if the timezone is missing from the publication date string?